packages/server/src/socket/manager.ts

```diff
@@ -136,11 +136,11 @@ export class SocketManager {
   }
 
   async setupSocket(socket: Socket.Socket) {
-    socket.on('message', async (data) => {
-      await this.handleSocketMessage(socket, data)
+    socket.on('message', (data) => {
+      void this.handleSocketMessage(socket, data)
     })
-    socket.on('disconnect', async () => {
-      await this.handleSocketDisconnect(socket)
+    socket.on('disconnect', () => {
+      void this.handleSocketDisconnect(socket)
```

Comment:
Why void?

Reply:
The `on` handler isn't async, so I put void here to indicate cleary that these are floating promises, and that the code needs to be correctly try catched otherwise it crashes the whole server